src/validation.cpp

```diff
@@ -1735,8 +1735,14 @@ void static FlushBlockFile(bool fFinalize = false)
 {
     LOCK(cs_LastBlockFile);
 
+    // we want to finalize flush the previous undo file, as it will have been written to
+    // for awhile even after we've moved onto the next blk file; if we don't do this, we
+    // end up with a re-opened but unflushed rev file, resulting in pre-allocated but not
+    // returned disk space allocation
+    int last_undo_file = fFinalize && nLastBlockFile > 0 ? nLastBlockFile - 1 : nLastBlockFile;
```

Comment:
```suggestion
    int previous_undo_file = fFinalize && nLastBlockFile > 0 ? nLastBlockFile - 1 : nLastBlockFile;
```
Given the comment on line 1738, is _previous_ more appropriate than last?

Reply:
Makes sense.